apps/src/turtle/skins.js

```diff
@@ -20,6 +20,12 @@ exports.load = function (assetUrl, id) {
       // slider speed gets divided by this value
       speedModifier: 10,
       turtleNumFrames: 10,
+      avatarSettings: {
+        width: 73,
+        height: 100,
+        numHeadings: 36,
+        image: new Image(),
```

Comment:
is there a specific reason this declaration was moved here rather than staying in turtle? All the others make sense, but I don't love the idea of creating a new Image in a config file